web/src/beta/lib/core/StoryPanel/Block/builtin/Camera/Editor.tsx

```diff
@@ -0,0 +1,174 @@
+import { useReactiveVar } from "@apollo/client";
+import { debounce } from "lodash-es";
+import { useContext, useMemo, useState } from "react";
+
+import Button from "@reearth/beta/components/Button";
+import CameraField from "@reearth/beta/components/fields/CameraField";
+import ColorField from "@reearth/beta/components/fields/ColorField";
+import ListField from "@reearth/beta/components/fields/ListField";
+import TextField from "@reearth/beta/components/fields/TextField";
+import { Camera } from "@reearth/beta/lib/core/engines";
+import { useVisualizer } from "@reearth/beta/lib/core/Visualizer/context";
+import { useT } from "@reearth/services/i18n";
+import { currentCameraVar } from "@reearth/services/state";
+import { styled } from "@reearth/services/theme";
+
+import { BlockContext } from "../common/Wrapper";
+
+type CameraBlock = {
+  id: string;
+  title?: string;
+  color?: string;
+  bgColor?: string;
+  cameraPosition?: Camera;
+};
+
+export type Props = {
+  items: CameraBlock[];
+  onUpdate: (
+    id: string,
+    fieldId: keyof CameraBlock,
+    fieldType: "string" | "camera",
+    value: string | Camera,
+  ) => void;
+  onItemRemove: (id: string) => void;
+  onItemAdd: () => void;
+  onItemMove: ({ id }: { id: string }, index: number) => void;
+  inEditor: boolean;
+};
+
+const CameraBlockEditor: React.FC<Props> = ({
+  items,
+  onUpdate,
+  onItemRemove,
+  onItemAdd,
+  onItemMove,
+  inEditor,
+}) => {
+  const t = useT();
+  const context = useContext(BlockContext);
+  const [selected, setSelected] = useState(items[0]?.id);
+
+  const visualizer = useVisualizer();
+  const currentCamera = useReactiveVar(currentCameraVar);
+  const handleFlyTo = useMemo(() => visualizer.current?.engine.flyTo, [visualizer]);
```

Comment:
Can you make the dep more specific so it will update less often (or in this case potentially not at all) 🙏🏻 
`[visualizer.current?.engine.flyTo]`

Reply:
This throws an error 
![image](https://github.com/reearth/reearth/assets/20891087/bd533455-fc98-4a93-818f-2f9d59a9a3d1)